### PR TITLE
chore(PPDSC-2234): investigate Storybook analytics

### PR DESCRIPTION
**Pull request by mutebg**
PPDSC-2234

**What**

1. Background - we need a way to track our storybook
2. What did you do:
2.1. Created a storybook plugin which utilises our Tealium and Consent component so we can track
2.2. Fixed Tealium component to take reactHelmet prop as its described in the docs.
4. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2234-storybook-analytics
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2234-storybook-analytics/storybook

File: .storybook/addons/tealium/register.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    context: {
+      url: path,
+      pageName,
+    },
+    originator: 'page-load',
+    trigger: 'load',
+  };
+
+  if (window && window.tealiumTrack) {
+    console.log('SEND_EVENT', event);
```
Comment (Vanals) [inline .storybook/addons/tealium/register.js:31]: left over console log?

File: .storybook/addons/tealium/register.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    window.tealiumTrack(event);
+    // window.utag.view(e);
```
Comment (Vanals) [inline .storybook/addons/tealium/register.js:33]: leftover comment

File: .storybook/addons/tealium/register.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+const CONFIG = {
+  privacyManagerId: '407619',
+  tealium: {
+    accountId: 'newsinternational',
+    profileId: 'thetimes.storybook',
+    env: isProduction ? 'prod' : 'dev',
+  },
+  consent: {
+    accountId: 259,
+    //TODO: change to https://storybook.newskit.co.uk
```
Comment (Vanals) [inline .storybook/addons/tealium/register.js:51]: left todo?

**Review by evgenitsn — APPROVED** [review]
Looks good, except the comments from Marco.

File: .storybook/addons/tealium/register.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      </ConsentSettingsLink>
+
+      <Tealium {...CONFIG.tealium} reactHelmet={Helmet} />
+      <Consent sourcePointConfigUnified={CONFIG.consent} reactHelmet={Helmet} />
+    </NewsKitProvider>
+  );
+};
+
+addons.register(ADDON_ID, api => {
+  // The addon only sets the
```
Comment (mstuartf) [inline .storybook/addons/tealium/register.js:77]: Incomplete comment

File: .storybook/addons/tealium/register.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  addons.add(TOOL_ID, {
+    type: types.TOOLEXTRA,
+    title: 'Privacy',
+    render: Tool,
+  });
+
+  // send initial page view event, since storybook doesn't do it via STORY_CHANGED
+  // When the preview boots, the first story is chosen via a selection specifier
+  api.on(STORY_SPECIFIED, () => {
+    console.log('STORY_SPECIFIED');
```
Comment (mstuartf) [inline .storybook/addons/tealium/register.js:88]: Do we need the console logs?

**Review by baburay23 — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -95,7 +95,10 @@
     "@storybook/addon-essentials": "^6.5.9",
     "@storybook/addon-links": "^6.5.9",
     "@storybook/addon-storysource": "^6.5.9",
+    "@storybook/addons": "^6.5.12",
+    "@storybook/components": "^6.5.12",
     "@storybook/react": "^6.5.9",
+    "@storybook/theming": "^6.5.12",
```
Comment (Xin00163) [inline package.json:101]: Do we need this?

Comment (mutebg) [reply package.json:101]: Maybe not in this PR ( I copied a bit of dependencies from Evgnies one so even don't need it now it needs to be installed later

**Review by Xin00163 — APPROVED** [review]

**Pull request merged**